syntax = "proto3";

import "google/protobuf/any.proto";

message CommandContainer {
	google.protobuf.Any command = 1;
}
